src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -529,6 +528,7 @@ final void doExecutePreparedStatement(PrepStmtExecCmd command) throws SQLServerE
         }
 
         String dbName = connection.getSCatalog();
+        boolean needsPrepare = false;
```

Comment:
Let's make the default `true` to be safe (better prepare than not). Should not matter but just to be a bit defensive...

Reply:
Otherwise things look good :-)

Reply:
Modified to true.